addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -200,6 +195,7 @@ def _create_invoices(self, sale_orders):
                 self.company_id.sale_down_payment_product_id = self.env['product.product'].create(
                     self._prepare_down_payment_product_values()
                 )
+                self.company_id.sale_down_payment_product_id.product_tmpl_id.active = False  # hide down payment product from user
```

Comment:
already done in _prepare_down_payment_product_values()

Reply:
> already done in _prepare_down_payment_product_values()

The product.product is set to active false, but the product.template is not at that point